src/main/java/testapp/modules/AppModule.java

```diff
@@ -0,0 +1,12 @@
+package testapp.modules;
+
+import com.google.sitebricks.SitebricksModule;
+import testapp.adapter.http.UserService;
+
+public class AppModule extends SitebricksModule {
+
+  @Override
+  protected void configureSitebricks() {
+    at("/v1/users").serve(UserService.class);
```

Comment:
shouldn't you install the repository module here so that your service can use it?